src/lando/pushlog/pushlog.py

```diff
@@ -133,9 +133,17 @@ def record_push(self) -> Optional[Push]:
             push.commits.add(commit)
 
         push.save()
+        self.push = push
```

Comment:
How is this value used?

Reply:
Apparently it no longer is. It was at some point in the past.